src/root/App/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from '../../assets/logo.svg'
 import './App.css'
+// import { Routes, Route } from 'react-router-dom'
+// import { Routes, Route, Navigate } from 'react-router-dom'
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom'
+import CaregiverDashboard from '../../layouts/caregiverDashboard/caregiverDashboard'
+import StressPatientDashboard from '../../layouts/stressPatientDashboard/stressPatientDashboard'
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <Routes>
+          <Route exact path="/" element={<CaregiverDashboard />} />
+          <Route exact path="/stress" element={<StressPatientDashboard />} />
```

Comment:
I haven't been able to test this myself, but is it possible to omit the element tag here? 

```suggestion
          <Route exact path="/" <CaregiverDashboard /> />
          <Route exact path="/stress" <StressPatientDashboard /> />
```